packages/esm-service-queues-app/src/patient-queue-metrics/metrics-header.component.tsx

```diff
@@ -1,15 +1,18 @@
 import React, { useState } from 'react';
+import { PatientBannerActionsMenu } from '@openmrs/esm-framework';
 import { useTranslation } from 'react-i18next';
 import { MessageQueue, ArrowRight } from '@carbon/react/icons';
-import { Button } from '@carbon/react';
-import { navigate, UserHasAccess } from '@openmrs/esm-framework';
+import { Button, ComboButton, MenuItem } from '@carbon/react';
+import { navigate, UserHasAccess, useLayoutType } from '@openmrs/esm-framework';
 import { spaBasePath } from '../constants';
 import { SearchTypes } from '../types';
 import PatientSearch from '../patient-search/patient-search.component';
 import styles from './metrics-header.scss';
 
 const MetricsHeader = () => {
   const { t } = useTranslation();
+  const isTablet = useLayoutType() === 'tablet';
+  const isPhone = useLayoutType() == 'phone';
```

Comment:
Nit: I don't know that we cater to the phone viewport. Unless the associated breakpoints represent dimensions matching those of tablets in use in facilities. If not, maybe `tablet` alone is sufficient.

Reply:
from experience, we do have a few of data collection  tablets that are relatively smaller compared to standard Tablet.

Reply:
@Murithijoshua Those would technically fall into the "unsupported" category.